app/src/test/java/com/farmerbb/taskbar/ui/TaskbarControllerTest.java

```diff
@@ -182,32 +173,17 @@ public void testGetTaskbarLayoutId() {
     }
 
     private int callGetTaskbarLayoutId(String taskbarPosition) {
-        return callInstanceMethod(
-                uiController,
-                "getTaskbarLayoutId",
-                from(String.class, taskbarPosition)
-        );
+        uiController.getTaskbarLayoutId(taskbarPosition);
     }
 
     private int callGetTaskbarGravity(String taskbarPosition) {
-        return callInstanceMethod(
-                uiController,
-                "getTaskbarGravity",
-                from(String.class, taskbarPosition)
-        );
+        uiController.getTaskbarGravity(taskbarPosition);
```

Comment:
return?

Reply:
Sorry about that, I did this late last night where I live and kinda rushed this.